tests/test_adresse.py

```diff
@@ -27,7 +27,16 @@ def test_serialization_only_required_fields(self):
             jdkwargs={"ensure_ascii": False},
         )
 
-        assert "DE" in address_json
+        assert "Nördliche Münchner Straße" in address_json
+        assert "27A" in address_json
+        assert "82031" in address_json
```

Comment:
Warum nochmal war die Assertion auf den Landescode nicht mehr wichtig? Ich weiß es wirklich nicht mehr, ob wir da irgendein Problem hatten oder ist das einfach nur durchgerutscht?

Reply:
Ich hatte das damals eingefügt um den default-wert für den Landescode in der Adresse zu testen.
Können es nochmal einfügen

Reply:
Na, wenn es sonst nirgends getestet wird... warum nicht

Reply:
danke